README.md

```diff
@@ -1,24 +1,175 @@
-# README
+# Gaming_console_reservation_back_end
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍